Rakefile

```diff
@@ -0,0 +1,29 @@
+require "yast/rake"
+
+Yast::Tasks.configuration do |conf|
+  # lets ignore license check for now
+  conf.skip_license_check << /.*/
```

Comment:
yeah, but this one actually skip license check for all files. which is probably what we do not want. So to try it, simply remove this line and run `rake check:license`